de/manual/ch_GUIMenus.xml

```diff
@@ -6665,459 +6665,1082 @@ url="&url-docs-de;guide/chapter_txns.html#txns-regstyle1"> &app; Kurs und Konzep
-              <entry>
-                <para>Schaltfläche
-                </para>
-              </entry>
+    <sect2 id="Gui-business-invoice">
+      <title>Fensteransicht für Rechnungsfunktionen</title>
+      
+      <figure pgwide="1">
+        <title>Rechnung bearbeiten</title><screenshot>
+        <screeninfo>1431x776x256</screeninfo>
+          <mediaobject>
+            <imageobject role="html">
+              <imagedata fileref="figures/GUI_Business_Invoice.png" srccredit="Christian Wehling" width="&img-w;" />
+            </imageobject>
 
-              <entry align="center">
-                <para>Beschreibung
-                </para>
-              </entry>
-            </row>
-          </thead>
+            <imageobject role="fo">
+              <imagedata fileref="figures/GUI_Business_Invoice.png" srccredit="Christian Wehling" />
+            </imageobject>
 
-          <tbody>
-            <row>
-              <entry>
-                <para><guibutton>Neu</guibutton>
-                </para>
-              </entry>
+            <caption>
+              <para>Die Ansicht zur Bearbeitung von <emphasis>Rechnungen</emphasis> und
+                    <emphasis>Lieferantenrechnungen</emphasis>.
+              </para>
+            </caption>
+          </mediaobject>
+        </screenshot>
+      </figure>
 
-              <entry>
-                <para>Den Dialog zum Anlegen eines neuen Kunden öffnen.
-                </para>
-              </entry>
-            </row>
+    <sect3 id="Gui-business-invoice-display">
+      <title>Anzeigebereich</title>
 
-            <row>
-              <entry>
-                <para><guibutton>Bearbeiten</guibutton>
-                </para>
-              </entry>
+      <para>In dieser Ansicht können Sie sowohl <emphasis>(Kunden-)Rechnungen</emphasis> und
+        <emphasis>Lieferantenrechnungen</emphasis> als auch <emphasis>Auslagenerstattungen</emphasis>
+        anlegen und bearbeiten.
+      </para>
 
-              <entry>
-                <para>Die Angaben des ausgewählten Kunden bearbeiten.
-                </para>
-              </entry>
-            </row>
+      <para>Wenn Sie mittels &mc.bus.cus.new-inv; oder &mc.bus.ven.new-bil; eine neue
+        Rechnung anlegen oder auch mit &mc.bus.emp.new-exp-vou; die Aufwendungen eines Mitarbeiters bezahlen,
+        dann öffnet sich im zweiten Schritt ein neues Fenster, in dem
+        Sie die einzelnen Rechnungspositionen erfassen können. Die Suchfunktionen
+        &mc.bus.cus.find-inv;, &mc.bus.ven.find-bil; sowie &mc.bus.emp.find-exp-vou;
+        sind eine Möglichkeit, die gewünschten Unterlagen zu finden.
+      </para>
 
-            <row>
-              <entry>
-                <para><guibutton>Neue Rechnung</guibutton>
-                </para>
-              </entry>
+      <para>Der obere Bereich im Fenster zeigt für die geöffnete Rechnung, von links
+        nach rechts, Rechnungs-Informationen, Abrechnungsdaten und Bemerkungen, im
```

Comment:
Ich glaube, die Bemerkungen sind intern, erscheinen also nicht in der Rechnung. Bitte prüfen und falls zutreffend reinschreiben, zumindest in der Feldbeschreibung.

Reply:
@fellen:
Danke für den Hinweis, die ausführliche Feldbeschreibung einer Kundenrechnung kommt irgendwo im Kapitel 7 "Business"